src/Microsoft.AspNetCore.Razor.Language/RazorParserFeatureFlags.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal abstract class RazorParserFeatureFlags
```

Comment:
This doesn't really need to be abstract if its internal

Reply:
I didn't want this to be constructible without using the Create method.